channeld/channeld.c

```diff
@@ -4633,12 +4640,10 @@ static void check_current_dataloss_fields(struct peer *peer,
 		     next_revocation_number);
 	if (next_revocation_number == 0)
 		memset(&old_commit_secret, 0, sizeof(old_commit_secret));
-	else {
+	else
+	{
```

Comment:
small nit: extra new line between `alse` and `{` here

Reply:
fixed